src/Pages/Login/LoginTap/AdminLogin/AdminLogin.js

```diff
@@ -0,0 +1,71 @@
+import React, { useState } from 'react';
+import { Link } from 'react-router-dom';
+import { useNavigate } from 'react-router-dom';
+import './AdminLogin.scss';
+
+const AdminLogin = () => {
+  const navigate = useNavigate();
+  const [id, setId] = useState('');
+  const [pw, setPw] = useState('');
+
+  const saveUserId = e => {
+    setId(e.target.value);
+  };
+
+  const saveUserPW = e => {
+    setPw(e.target.value);
+  };
+
+  const goToMain = e => {
+    e.preventDefault();
+
+    fetch('https://f29c-211-106-114-186.jp.ngrok.io/user/signup', {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json;charset=utf-8',
+      },
+      body: JSON.stringify({
+        text: id,
+        password: pw,
+      }),
```

Comment:
```suggestion
      body: JSON.stringify(유저 정보 하나로 관리한 state),
```
이 부분에 일일히 넣어주지 않아도 이와같이 적용할 수 있습니다.